releasenotes/notes/asgi-support-websockets-f970956661062882.yaml

```
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    asgi: Trace websockets in asgi middleware behind feature flag. Set ``DD_TRACE_ASGI_WEBSOCK=true`` to enable.
```

```suggestion
    asgi: Trace websockets in asgi middleware behind feature flag. Set ``DD_ASGI_TRACE_WEBSOCKET=true`` to enable.
```
I missed this typo on the previous review, sorry about that.